x-pack/plugin/src/test/resources/rest-api-spec/test/ilm/40_explain_lifecycle.yml

```diff
@@ -114,6 +114,7 @@ teardown:
   - match: { indices.my_index.step: "complete" }
```

Comment:
Can you add some `is_true/match` checks in here that the `phase_execution` is in the format we expect?

Reply:
the problem here is that phase_execution is currently set to be nothing for the complete phase. 

that is why there is the check below for

```
- is_false: indices.my_index.phase_execution
```

I will move the IndexLifecycleIT client test that verifies the format we expect to plugin:ilm.

Reply:
Can we construct a policy that moves an index into having 100 replicas, then we know it'll be stalled in a particular place, would that work? (might have timing issues)

Reply:
we can, and that is what I will do. I just saw the changes I needed to make to the client integration tests, and it was an easy win there. I realize, now, that was lazy thinking since we shouldn't rely on client tests to catch server problems